integrations/azure-devops/.port/resources/blueprints.json

```diff
@@ -25,6 +25,10 @@
           "default": 0,
           "type": "number"
         },
+        "defaultTeam": {
+          "type": "string",
+          "description": "Default Team of the project"
+        },
         "slack": {
```

Comment:
add title and also for all others that their name is combined from multiple words, such as `minimumApproverCount` etc..